sw/ground_segment/python/svinfo/svinfoviewer.py

```diff
@@ -122,6 +122,9 @@ def __init__(self):
         self.Bind(wx.EVT_SIZE, self.OnSize)
         self.Bind(wx.EVT_CLOSE, self.OnClose)
 
+        ico = wx.Icon(os.getenv("PAPARAZZI_HOME") + "/sw/ground_segment/python/svinfo/gps.ico", wx.BITMAP_TYPE_ICO)
```

Comment:
Could you make this more robust if paparazzi env are not set ? Something like https://github.com/paparazzi/paparazzi/blob/master/sw/ground_segment/python/settings_app/settingsapp.py#L10
Also, it should be PAPARZZI_SRC, same for most of the other pull requests.